server/src/main/java/com/cloud/uuididentity/UUIDManagerImpl.java

```diff
@@ -119,7 +119,7 @@ public boolean IsUuidFormat(String uuid) {
         }
         Identity identity = (Identity) this._entityMgr.findById(entityType, customId);
         if (identity == null) {
-            throw new InvalidParameterValueException("Unable to find UUID for id " + customId);
+            throw new InvalidParameterValueException(String.format("Unable to find UUID for id %s of type %s", customId, entityType.getSimpleName()));
```

Comment:
could entityType be null?

Reply:
it could but it would have thrown an error in line 120 I thought about
```suggestion
            throw new InvalidParameterValueException(String.format("Unable to find UUID for id %s of type %s",
                                                                   customId,
                                                                   (entityType == null ? "<unknown type>" ? entityType.getSimpleName())));
```

I wasn't sure if it was needed.

Reply:
@DaanHoogland If entityType is null, the resultant dao is null and can be a potential NPE. May be this extra null check here can guard, and you can keep the entity type in error message.

```
if (customId == null || entityType == null) {
    return null;
}
```